modules/command-engine/core/src/test/scala/surge/core/AggregateRefTraitSpec.scala

```diff
@@ -15,7 +16,17 @@ import surge.internal.persistence.{ AggregateRefTrait, PersistentActor }
 
 import scala.concurrent.Future
 
-class AggregateRefTraitSpec extends TestKit(ActorSystem("AggregateRefTraitSpec")) with AsyncWordSpecLike with Matchers with ScalaFutures {
+class AggregateRefTraitSpec
+    extends TestKit(ActorSystem("AggregateRefTraitSpec"))
+    with AsyncWordSpecLike
+    with Matchers
+    with ScalaFutures
+    with BeforeAndAfterAll {
+
+  override def afterAll(): Unit = {
+    TestKit.shutdownActorSystem(system)
```

Comment:
Can we .shutdownActorSystem(system, verifySystemShutdown = true) so that we can get a more eye catching error ?

Reply:
Yep - will do